walk/config.go

```diff
@@ -33,14 +33,27 @@ import (
 	gzflag "github.com/bazelbuild/bazel-gazelle/flag"
 )
 
+// GenerationModeType represents one of the generation modes.
+type GenerationModeType string
+
+// Generation modes
+const (
+	// Update: update and maintain existing BUILD files
+	GenerationModeUpdate GenerationModeType = "update"
+
+	// Create: create new and update existing BUILD files
+	GenerationModeCreate GenerationModeType = "create"
```

Comment:
What do you think about `update_only` and `create_and_update`? Slightly more verbose, but also more descriptive.